src/Application/Projects/Commands/DeleteProject/DeleteProjectCommand.cs

```diff
@@ -0,0 +1,26 @@
+﻿using CoduTeam.Application.Common.Interfaces;
+
+namespace CoduTeam.Application.Projects.Commands.DeleteProject;
+
+public record DeleteProjectCommand(int Id) : IRequest;
+
+public class DeleteProjectCommandHandler(IApplicationDbContext dbContext) : IRequestHandler<DeleteProjectCommand>
+{
+    public async Task Handle(DeleteProjectCommand request, CancellationToken cancellationToken)
+    {
+        var entity = await dbContext.Projects.
+            Where(i => i.Id == request.Id)
+            .SingleOrDefaultAsync(cancellationToken);
```

Comment:
Let it be FirstOrDefault, it will be better in case of performance, and due to primary key violation it can't be more then 1 item